### PR TITLE
[OVA]: Minor change to remove hostname from curl

### PR DESCRIPTION
**Pull request by mukul-seagate11**
### Describe your changes in brief
Minor change to remove Hostname from curl

### Changes
 - [ ] Why is this change required? What problem does it solve?
 Minor change to remove hostname from curl as hostname is not being ping from other setup during CSM endpoint validation

 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [x] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [x] tested locally
 - [ ] added new dependencies
 - [x] updated the docs
 - [ ] added a test

### CODE REVIEW

**Review by AshwiniS-Borse — APPROVED** [review]

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 1]
```diff
@@ -65,14 +65,16 @@ Procedure
...
-**********
-#. Validate the CSM endpoint using below command. Use hostname or ip of your OVA VM.
+**********************************
+
+#. Validate the CSM endpoint using below command. Use hostname or ipaddress as per your VM
    
    ::
    
-        curl --request POST '<https://$HOSTNAME OR https://$IPADDRESS>/api/v2/login?debug' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k
+      curl --request POST '<https://$IPADDRESS>:31169/api/v2/login?debug>' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k
```
Comment (shailesh-vaidya) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:76]: Can we use - `https://<$IPADDRESS>:31169/api/v2/login?debug` . As only IP Address will change based on setup. 
Also, we may dynamically populate it as below 
```
[root@test ~]# echo $(ifconfig eth1 | grep inet -w | awk '{print $2}')
192.168.49.191
```

Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst]: Done, changed accordingly

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 2]
```diff
@@ -72,10 +72,9 @@ Validate CSM endpoint via REST API
 #. Validate the CSM endpoint using below command. Use hostname or ipaddress as per your VM
    
    ::
-   
-      curl --request POST '<https://$HOSTNAME:31169/api/v2/login?debug>' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k
 
-      curl --request POST '<https://$IPADDRESS>:31169/api/v2/login?debug>' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k      
+      export IPADDRESS=`ifconfig eth1 | grep inet -w | awk '{print $2}'` && echo $IPADDRESS
+      curl --request POST 'https://$IPADDRESS>:31169/api/v2/login?debug' --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k
```
Comment (shailesh-vaidya) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:77]: Please remove `>`

**Comment by mukul-seagate11** [comment]
Raising new PR as DCO failed

**Pull request closed without merging**